src/PyCall.jl

```diff
@@ -298,6 +300,14 @@ function setindex!(o::PyObject, v, s::Union{Symbol,AbstractString})
     o
 end
 
+getproperty(o::PyObject, s::Symbol) = getindex(o, s)
+getproperty(o::PyObject, s::AbstractString) = getindex(o, s)
```

Comment:
Better to move the definition of `getindex` here, since `getindex` will be deprecated.

Reply:
Done.